src/openPr.js

```diff
@@ -80,6 +89,12 @@ module.exports = async function ({ context, inputs, packageVersion }) {
     throw new Error('packageVersion is missing!')
   }
 
+  const token = inputs['github-token']
+
+  const baseReleaseTag = inputs['base-tag']
+  if (baseReleaseTag) {
+    await fetchReleaseByTag(token, baseReleaseTag)
+  }
```

Comment:
not sure I understand what this does. why would invoking a function and doing nothing with the result achieve anything?

Reply:
ah I see what this does and @Ceres6 comment below. ok no we need to do this differently. either we assert the existence, as @Ceres6 suggested, or we actually retrieve the release object and pass it around. the second option is probably more work but I have a feeling it would be a better solution